packages/@aws-cdk/aws-ecs/test/ec2/integ.bottlerocket.ts

```diff
@@ -0,0 +1,19 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
```

Comment:
I wonder for the integ test if there's a way where we can validate this cluster is hooked up with the ec2 instance that running with bottlerocket AMI, since in the current integ test we just validate the CFN deployment is successful or not. Might be overkill still given that we don't do it with the other integ test either.

Reply:
I would make a unit test to validate that stack with the cluster with instances running the bottlerocket AMI will always have a LaunchConfiguration resource where the AMI comes from correct parameter store.

For the integ testing, we probably just need to make sure it deploys correctly as the snapshot json.

What do you think?

Reply:
@iamhopaul123 

Just added two unit tests here. Please check it out and see if it makes sense.

https://github.com/aws/aws-cdk/blob/db78b1ea6e1994518159518776ecfb38bad12940/packages/%40aws-cdk/aws-ecs/test/test.ecs-cluster.ts#L1478-L1591

Reply:
:+1: